mozperftest_tools/mozperftest_tools/side_by_side.py

```diff
@@ -148,20 +148,23 @@ def generate_step_chart(self, oldvid, newvid, vismetPath, prefix, metric, output
         plt.savefig(str(output / "%s-%s-step.png" % (prefix.rstrip("_"), metric)))
         plt.clf()
 
-    def _search_for_paths(self, rev_ids, artifact, open_data=False):
+    def _search_for_paths(self, rev_ids, artifacts, open_data=False):
```

Comment:
Could we revert the changes to `_search_for_paths` given that we no longer need the multi-artifact functionality? The alternative is that we'll need to modify this to properly organize the artifacts we are gathering into a dictionary.